core/src/main/java/io/undertow/protocols/ajp/AjpClientChannel.java

```diff
@@ -18,29 +18,28 @@
 
 package io.undertow.protocols.ajp;
 
-import static io.undertow.protocols.ajp.AjpConstants.FRAME_TYPE_END_RESPONSE;
```

Comment:
Caused by IDE automatic organization of imports. In undertow there is not common standard how imports are organized and the ordering might differ among the classes. If it is an issue, I can rearange them back.